pkg/messaging/pair/client.go

```diff
@@ -0,0 +1,83 @@
+package pair
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+
+	"go.nanomsg.org/mangos/v3/protocol/pair"
+	"log"
+)
+
+func StartMessagingPairClient(ctx context.Context, nanomsgURL string, requestPair chan RequestPair, responsePair chan ResponsePair) error {
```

Comment:
Create new context with cancel from given one. Add cancel call in case of error. It can be useful if any error happens. (And do not forget to call cancel with defer in goroutine below)